src/selectors/entities/preferences.ts

```diff
@@ -8,7 +8,7 @@ import {General, Preferences} from '../../constants';
 import {getConfig, getLicense} from 'selectors/entities/general';
 import {getCurrentTeamId} from 'selectors/entities/teams';
 
-import {PreferenceType} from 'types/preferences';
+import {PreferenceType, Theme} from 'types/preferences';
```

Comment:
Why this changes appear here? Looks like a merge conflict that is resolved reverting changes.

Reply:
@jespino I'm using the `getTheme` selector in a small part the main feature, and I wanted to cleanup the types around it as it's currently typed as returning `any`. Having clearer types during development helps me to be more sure, but by all means, it could be separated in another PR.